src/app/components/BookFilters/Filter.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  CheckSoloIcon,
+  DotsIcon,
+} from 'dgx-svg-icons';
+
+class Filter extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selected: false,
+      buttonActive: false,
+    };
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onClick() {
+    const filterId = this.props.filter.toLowerCase().split(' ').join('-');
+    this.props.onClick(filterId, !this.state.selected);
+    this.setState({
+      selected: !this.state.selected,
+      buttonActive: true,
+    });
+
+    setTimeout(() => {
+      this.setState({ buttonActive: false });
+    }, 1000);
+  }
+
+  render() {
+    const {
+      selected,
+      buttonActive,
+    } = this.state;
+    const filter = this.props.filter;
+    const active = selected ? 'active' : '';
```

Comment:
I would give it `activeClass` name to be clearer it's a class.

Reply:
👌